docs/modules/qbi-helper/intro-to-qbi.md

```diff
@@ -0,0 +1,55 @@
+<%(TOC)%>
+
+# Introduction to Query-based interest
+
+<%(Callout message="
+Before reading this document, make sure you are familiar with:
+
+  * [Query-based interest](https://docs.improbable.io/reference/latest/shared/reference/query-based-interest)
+  * [Workers in the GDK]({{urlRoot}}/reference/concepts/worker)
+")%>
+
+The Query-based interest helper module offers 3 classes to help construct an `Interest` component:
+
+* [`InterestTemplate`]({{urlRoot}}/api/query-based-interest/interest-template), to define and manipulate an `Interest` component.
+* [`InterestQuery`]({{urlRoot}}/api/query-based-interest/interest-query), an interest query with methods to easily set a [`Constraint`]({{urlRoot}}/api/query-based-interest/constraint) and define what components the query should return.
+* [`Constraint`]({{urlRoot}}/api/query-based-interest/constraint), a static class with constructors for all the possible constraints an [`InterestQuery`]({{urlRoot}}/api/query-based-interest/interest-query) can have.
+
+To understand how these classes can be used, this document provides an overview of key primitives in Query-based interest and how they link together.
```

Comment:
> To understand how these classes can be used, this document provides an overview of key primitives in Query-based interest and how they link together.

This page provides a quick overview of the key primitives in Query-based interest.